pkg/controller/realmadmin/settings_modify.go

```diff
@@ -242,6 +243,7 @@ func (c *Controller) HandleSettings() http.Handler {
 			} else {
 				currentRealm.AllowAdminUserReport = false
 			}
+			currentRealm.AllowUserReportWebView = form.AllowUserReportWebView
```

Comment:
Should this be moved up into the "AllowUserReport" block? It doesn't make sense to enable this if UserReport isn't also enabled.

Reply:
by keeping it separate - we can show errors if they try to enable one w/o the other (realm.BeforeSave)

Reply:
Should we change "AllowAdminUserReport" to behave the same? I don't have a strong opinion on which is the right way, but I think they should be the same

Reply:
yeah - I changed them to be the same.

Reply:
![image](https://user-images.githubusercontent.com/92319/120362009-6e4cc880-c2bf-11eb-9155-d7f8151d01e2.png)